README.md

```diff
@@ -1 +1,44 @@
-# SachBoxNew
\ No newline at end of file
+# SachBox
+
+## Description
+SachBox is an expansive game that connects communities who have been disconnected due to the recent pandemic through online party games.
+
```

Comment:
Can we also add a section on how to download and start up the project. 

Download
1) git clone 
2) npm install

Run
1) npm start
2) node server.js (in progress)

Reply:
Added the project instructions - will fix it up

Reply:
my bad, can we be more detailed looool
in the Git doc there is a section on `npm start` and `git clone`

for `node server.js` have a note saying you have to be in "my-app"

Reply:
Added more info, I do not know if that is what you wanted. If not, tell me and I'll update it.

Reply:
Git DOC NVM IGNORE ME, I AM BRAIN DEAD

Reply:
Added the info from the Git doc